tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -247,13 +240,15 @@ message PartitionDescProto {
   required string partitionName = 1;
   repeated PartitionKeyProto partitionKeys = 2;
   optional string path = 3;
```

Comment:
When adding or dropping partition, CatalogClient calls alter table API with AlterTableDescProto. and the proto maintains partition informations to PartitionDescProto. When adding partition, the partition path will be not null. But when dropping partition, the path will be null because Tajo can delete partition informs in CatalogStore with partition name.